app/src/main-process/main.ts

```diff
@@ -722,26 +726,19 @@ function createWindow() {
   const window = new AppWindow()
 
   if (__DEV__) {
-    const {
-      default: installExtension,
-      REACT_DEVELOPER_TOOLS,
-    } = require('electron-devtools-installer')
-
```

Comment:
@sfadschm We would prefer to only include this for dev builds as it is not needed for any other build.